stack-lts-13.0.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-12.26
+resolver: lts-13.0
```

Comment:
Is this the same GHC?

It's not a big deal, but just for context: the lowest resolver sets our lower bounds on publish as well as verifies CI against that LTS' GHC. So we always start out by picking the lowest LTS for a given GHC minor version, since that'll give us the lowest lower bounds we can. If you found there was some package there that messed up the build, moving only it up as an extra-dep would be "better" than moving to a higher resolver of the same GHC since that would move all the contained packages' bounds up, rather than just the one that was trouble.

Again, no big deal, but just that's the thinking going into the choice.

Reply:
Ah, then just remove this. The lts-14 config is already testing 8.6. Kind of a downside of using the generated matrix is that you lose those comments.

Reply:
Oh, but also you don't need to do this. If you spell it correctly, we have it back to 6.8 ;)

> 	6.8.1. **British spelling since 8.6.1**

Reply:
I wonder if you had it in your head you had to drop something GHC-related because you're doing work in the vicinity of (but not using) `DerivingVia`. So when your typo hit that error, you figured that was it :thinking:

Reply:
Still blocking on this. Even if you do want to drop 8.6 support, just remove the lts-12 file and leave testing 8.6 to the lts-14 file. No need to add lts-13; it'd be redundant.

```console
% stack ls-snapshots
-- snip --
lts-18.28,8.10
lts-16.31,8.8
lts-14.27,8.6
lts-12.26,8.4
-- snip --
```